action/protocol/execution/protocol_test.go

```diff
@@ -483,6 +484,7 @@ func (sct *SmartContractTest) prepareBlockchain(
 			sf,
 			protocol.NewGenericValidator(sf, accountutil.AccountState),
 		)),
+		blockchain.BlockTimeCalculatorBuilderOption(btcd),
```

Comment:
can do `BlockTimeCalculatorBuilderOption(testutil.DummyBlockTimeBuilder())` just like in all other tests? and remove L477

Reply:
BlockTimeCalculator has been deprecated